### DISCUSSION TITLE
Race Delta

### DISCUSSION THREAD

**Discussion by RapidDestroyer1** [theOehrly/Fast-F1#223] [Q&A] [answered]
Hi,
I am hoping to create a plot which shows the time difference (delta) between one driver and the other throughout an entire race. So I guess the x axis would be the lap number and the y axis the time between the 2 cars at the start of each lap. It would be something like the image attached created by Brrrakef1
Any advice would be greatly appreciated!
Thanks.
![Screenshot 2022-08-16 8 24 45 PM](https://user-images.githubusercontent.com/111387009/184964949-aafc3ec2-0756-49a4-a55c-3a7b47038cae.png)

**Comment by RapidDestroyer1** [comment, marked as answer]
No worries I've figured it out!
![2022HungarianGrandPrix-Race-WholeRacedata](https://user-images.githubusercontent.com/111387009/185357704-778495f1-c37b-4758-ba11-782442f7704e.png)

**Reply by harish5p** [reply]
Could you please share the code? It might help others too in the future
